app/api/helpers/custom_fields.py

```diff
@@ -245,3 +246,15 @@ def validate(self, value):
             self.validation_error = 'Value of %s is not in available choices'
             return False
         return True
+
+
+class Licence(CustomField):
+    """
+    Custom field that gets the licence details given the licence name
+    """
+    __schema_example__ = "{'name': 'Licence name' 'long_name': 'Long licence name', \
```

Comment:
The webapp requires the `long_name` field of the licence right now. But as we are using the db model of copyright in the api, on `long_name` field is present there. Also right now, the webapp requires the `long_name` of the license, in future other details about the license may also be required, better to provide all the details in the api :) .